delfin/cmd/task.py

```diff
@@ -46,18 +46,9 @@ def main():
     task_server = service.TaskService.create(binary='delfin-task',
                                              coordination=True)
     leader_election = service.LeaderElectionService.create()
-    metrics_task_server = service. \
```

Comment:
metric worker node need not start resource task and alert task.. if we start new service inside scheduler start we cant control the service start during multi process spawn

Reply:
Reversed